### PR TITLE
Removed addMetrics code in client library

### PR DESCRIPTION
**Pull request by JayMehta11**
In #1332 PR.

As per the discussion, addMetrics code has been removed from the client library.

### CODE REVIEW

File: clientlibs/js/src/UpGradeClient/UpgradeClient.ts [hunk 1]
```diff
@@ -475,7 +473,4 @@ export default class UpgradeClient {
    * This feature is available but not recommended for use as it is not fully regression tested in recent releases.
```
Comment (danoswaltCL) [inline clientlibs/js/src/UpGradeClient/UpgradeClient.ts:473]: comment can be deleted too

Comment (JayMehta11) [reply clientlibs/js/src/UpGradeClient/UpgradeClient.ts]: Code is updated

**Comment by VivekFitkariwala** [comment]
Hold on to this PR. I have added a [question](https://github.com/CarnegieLearningWeb/UpGrade/issues/1332#issuecomment-2172474351) to the ticket.

**Comment by JayMehta11** [comment]
Due to merge conflicts, I am closing this PR. A new PR has been created to address these issues and continue the work. Please refer to the new PR for the latest updates: #1662.

**Pull request closed without merging**